eng/performance/sdk_scenarios.proj

```diff
@@ -82,10 +82,10 @@
 
   <ItemGroup>
     <HelixWorkItem Include="@(SDKWorkItem -> '%(Identity) Clean Build')">
-      <Command>%(Command) clean_build</Command>
+      <Command>%(Command) clean_build --scenario-name &quot;%(Identity)&quot;</Command>
```

Comment:
Does `%(Identity)` here become `%(Identity) Clean Build` automatically or should it be explicitly included?

Reply:
It includes ```Clean Build```